### PR TITLE
chore: format integration tests

### PR DESCRIPTION
**Pull request by TomAFrench**
# Description

## Problem\*

Resolves 

## Summary\*

This PR runs the formatter over the integration test suite to flag up any issues.

1. The nested for-loops in `6_array` causes a panic due to start>end when slicing.
2. Due to not handling all statements/expressions, we're getting some funky indentation. (see `1327_concrete_in_main`)

## Documentation

- [ ] This PR requires documentation updates when merged.

  

  - [ ] I will submit a noir-lang/docs PR.

  

  - [ ] I will request for and support Dev Rel's help in documenting this PR.

  
  

## Additional Context

# PR Checklist\*

- [ ] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: tooling/nargo_cli/tests/execution_success/simple_shield/src/main.nr [hunk 1]
```diff
@@ -13,7 +13,8 @@ fn main(
   // Receiver public key
   to_pubkey_x: Field, 
   to_pubkey_y: Field, 
-) -> pub [Field; 2] {
+) -> pub [Field; 2]
+{
```
Comment (TomAFrench) [inline tooling/nargo_cli/tests/execution_success/simple_shield/src/main.nr:17]: This seems like a bit of an odd choice.

File: tooling/nargo_cli/tests/execution_success/simple_bitwise/src/main.nr [hunk 2]
```diff
@@ -5,5 +5,6 @@ fn main(a: bool, b: bool, c: u8, d: u8) -> pub u8 {
     let x = c & d;
     let y = c ^ d;
     let z = c | d;
-    (i as u8) + (j as u8) + (k as u8) + x + y + z
+    (
+    i as u8 + j as u8 + k as u8 + x + y + z
```
Comment (TomAFrench) [inline tooling/nargo_cli/tests/execution_success/simple_bitwise/src/main.nr:9]: Not sure I like the removal of parentheses here. Also one is getting stranded on the previous line.

File: tooling/nargo_cli/tests/execution_success/scalar_mul/src/main.nr [hunk 3]
```diff
@@ -11,7 +11,7 @@ fn main(
      let mut priv_key = a;
      let mut pub_x: Field = a_pub_x;
      let mut pub_y: Field = a_pub_y;
-     if a != 1 { // Change `a` in Prover.toml to test input `b`
+    if a != 1 { // Change `a` in Prover.toml to test input `b`
```
Comment (TomAFrench) [inline tooling/nargo_cli/tests/execution_success/scalar_mul/src/main.nr:14]: This is correct, the other lines are indented by 5

File: tooling/nargo_cli/tests/execution_success/integer_array_indexing/src/main.nr [hunk 4]
```diff
@@ -7,6 +7,5 @@ fn main(arr: [Field; ARRAY_LEN], x: u32) -> pub Field {
     value += arr[0 as u32];
     value += arr[1 as Field];
 
-    value + (x as Field)
-
+    value + x as Field
```
Comment (TomAFrench) [inline tooling/nargo_cli/tests/execution_success/integer_array_indexing/src/main.nr:10]: again, lack of parentheses make this ambiguous imo.

**Comment by TomAFrench** [comment]
fyi @f01dab1e (Thank you for the work on the formatter btw!)

File: tooling/nargo_cli/tests/execution_success/6_array/src/main.nr
Comment (TomAFrench) [inline tooling/nargo_cli/tests/execution_success/6_array/src/main.nr:1]: "Test 6" in this file panics. Looks to be something to do with nested for-loops

File: tooling/nargo_cli/tests/execution_success/simple_shield/src/main.nr (hunk 1 above)
Comment (vezenovm) [reply tooling/nargo_cli/tests/execution_success/simple_shield/src/main.nr:17]: I agree. Could we change this in the formatter?

File: tooling/nargo_cli/tests/execution_success/simple_bitwise/src/main.nr (hunk 2 above)
Comment (vezenovm) [reply tooling/nargo_cli/tests/execution_success/simple_bitwise/src/main.nr:9]: I agree this is broken imo. The stranded paren just adds confusion.

**Pull request closed without merging**